spec/duration.html

```diff
@@ -1586,18 +1604,39 @@ <h1>
         </dd>
       </dl>
       <emu-alg>
+        1. Let _didExpandCalendarUnit_ be *false*.
+        1. Let _date_ be ? ToTemporalDate(_isoDateTime_.[[ISODate]]).
```

Comment:
This line doesn't work - ToTemporalDate takes an ECMAScript language value, and ISO Date Record is a spec value. It should be:
```suggestion
        1. Let _date_ be ! CreateTemporalDate(_isoDateTime_.[[ISODate]], _calendar_).
```

But, in the past we've tried to get rid of creating Temporal objects internally that aren't exposed to user code. So this makes me think we probably need to refactor CompareDurations. Maybe we need to pass _originEpochNs_, _isoDateTime_.[[ISODate]], _timeZone_ , and _calendar_ to get the same info that we'd otherwise get by passing in _zonedRelativeTo_ and _plainRelativeTo_.

This unfortunately probably also means refactoring DateDurationDays to take an ISO Date Record and a calendar type, but that seems like an improvement anyway.